app/controllers/api/host_initiator_groups_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  class HostInitiatorGroupsController < BaseController
+    def refresh_resource(type, id, _data = nil)
+      enqueue_action(type, id, "Refreshing", :method_name => :refresh_ems)
+    end
+
+    def create_resource(type, _id = nil, data = {})
+      raise BadRequestError, "ems_id not defined for #{type} resource" if data['ems_id'].blank?
+
+      ext_management_system = ExtManagementSystem.find(data['ems_id'])
+      task_id = HostInitiatorGroup.create_host_initiator_group_queue(session[:userid], ext_management_system, data)
```

Comment:
@gregoryb-at-ibm you should check that `ext_management_system.class_by_ems(:HostInitiatorGroup).supports?(:create)`

Reply:
`AuthKeyPairsController` is a good example, https://github.com/ManageIQ/manageiq-api/blob/master/app/controllers/api/auth_key_pairs_controller.rb

Reply:
And once that is in place can you add a spec that this fails for a provider that doesn't support creation of host_initiator_groups?  (You can use `stub_supports`)